client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -187,11 +187,34 @@ private void submitRetryPushData(
           new CelebornIOException("Revive Failed, remain revive times " + remainReviveTimes));
     } else if (mapperEnded(shuffleId, mapId, attemptId)) {
       logger.debug(
-          "Retrying push data, but the mapper(map {} attempt {}) has ended.", mapId, attemptId);
+          "Revive for push data success, but the mapper already ended for shuffle "
```

Comment:
let's use the "{}" style, as we discussed.

Reply:
> let's use the "{}" style, as we discussed.

How about current?